### ISSUE TITLE
some issues about code

### ISSUE THREAD

**Issue by jacksonsc007** [strongwolf/DW#19]
Hi, @strongwolf , there are some questions:
1. In this [line](https://github.com/strongwolf/DW/blob/27f50e5bf8d9ea021d5fe1a7a7e47e06bc5db208/dw_head.py#L178) ，p_pos_weight is normalized with difference µ whose value are 5 and 3. I wonder if there are some insights?
2. As illustrated in the code, IoU score is  represented as an expentional function of the reg loss. What is strangest to me is that loc_loss is further computed by a binary_cross_entropy loss even we already get reg_loss in this [line](https://github.com/strongwolf/DW/blob/27f50e5bf8d9ea021d5fe1a7a7e47e06bc5db208/dw_head.py#L185). Could you further explicate this?
Thanks a lot.

**Comment by jacksonsc007** [comment]
For question 2, I found that `loc_loss = F.binary_cross_entropy(p_loc, torch.ones_like(p_loc), reduction='none')` actually is equivalent to `5 * reg_loss` which is in accordance with the paper.

**Comment by strongwolf** [comment]
For q1, you can refer to this [issue](https://github.com/strongwolf/DW/issues/3). For q2, you are right. `loc_loss = F.binary_cross_entropy(p_loc, torch.ones_like(p_loc), reduction='none')` is equivalent to `5 * reg_loss`.

**Comment by jacksonsc007** [comment]
Sorry for my negligence in checking closed issue before proposing this one. Thanks for your reply.
